src/python/nimbusml/tests/pipeline/test_load_save.py

```diff
@@ -148,6 +150,12 @@ def test_model_datastream(self):
 
         os.remove(model_filename)
 
+    def test_mlnet_model_can_be_scored(self):
+        data = FileDataStream.read_csv(test_file, sep=',', numeric_dtype=np.float32)
```

Comment:
ML.NET model is trained on a file with label, and it expects label to be present in the schema of the data being passed to `predict`